hello/templates/pokemonGo.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>pokemon Go</title>
+     <link rel="stylesheet" href="/static/pikatchu.css">
```

Comment:
no właśnie te pliki statyczne. Używamy tutaj znacznika {% static %}, ale to jeszcze przed nami